web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js

```diff
@@ -135,30 +135,22 @@ export const MachineLearningSettings = () => {
         <Divider height={32} />
 
         {backends.length > 0 && (
-          <Description style={{ marginTop: 0, maxWidth: 680, lineHeight: "1.5em" }}>
-            A connected model has been detected! If you wish to fetch
-            predictions from this model, please follow these steps:
-            <br />
-            <br />
-            1. Navigate to the <i>Data Manager</i>.<br />
-            2. Select the desired tasks.<br />
-            3. Click on <i>Batch predictions</i> from the{' '}
-            <i>Actions</i> menu.
-            <br />
-            <br />
-            Additionally, you can configure the system to use this model for
-            fetching live predictions in {" "}
-            <a
-              style={{ color: "unset", textDecoration: "underline" }}
-              target="_blank" href="https://labelstud.io/guide/labeling.html#Choose-which-tasks-to-label"
-            ><i>Label&nbsp;All&nbsp;Tasks</i></a>{" "}
-            and{" "}
-            <a
-              style={{ color: "unset", textDecoration: "underline" }}
-              target="_blank" href="https://labelstud.io/guide/get_started#Interface"
-            ><i>Quick View</i></a>{" "}
-            on the <i>Annotation</i> tab.
-          </Description>
+          <>
+            <Description style={{ marginTop: 0, maxWidth: 680, lineHeight: "1.5em" }}>
+              A connected model has been detected! If you wish to fetch
+              predictions from this model, please follow these steps:
+              <br />
+              <br />
+              1. Navigate to the <i>Data Manager</i>.<br />
+              2. Select the desired tasks.<br />
+              3. Click on <i>Retrieve predictions</i> from the{' '}
+              <i>Actions</i> menu.
+            </Description>
+            <Description style={{ marginTop: 0, display: "flex", gap: "5px" }} className="description_alert_warning">
+              <IconInfo width="20" height="20" />
+              If you want to use the model predictions for prelabeling, please configure this in the Annotation settings.
```

Comment:
Link to Annotation settings would be helpful as we discussed

Reply:
Yes, I'm very supportive if you can implement that :)